docs/docs/rad-lab-ui/ui_installation/frontend.md

```diff
@@ -3,47 +3,6 @@ sidebar_position: 4
 title: 04 - Web Application
 ---
 
-import create from "zustand";
```

Comment:
Do we know what exactly was this script for at the first place ?

Reply:
I believe it was to take Project ID as input so that the scripts in the given page automatically gets replaced for the given project.

Was it even working before ?

Reply:
Yeah, Zustand is a lightweight store. It was used in v1 of MDX to take the project ID and populate it in several other parts. It was completely broken by Docusaurus' upgrade to v3 of MDX. I figured it was so niche (only this page) that it was better to simply remove it.

Reply:
Sounds good!